src/main/scala/com/workday/warp/common/HasCoreWarpProperties.scala

```diff
@@ -91,6 +91,16 @@ trait HasCoreWarpProperties extends WarpPropertyLike {
     */
   val WARP_LOG_FILE: PropertyEntry = PropertyEntry("wd.warp.log.file", isRequired = false, "build/warp.log")
 
+  /**
+    * Whether MeasurementCollectors should log stacktraces when errors occur, or just an error message.
+    *
+    * Note that setting this to true can result in an overly verbose build log.
+    *
+    * Required: No
+    * Default Value: false
+    */
+  val WARP_LOG_STACKTRACES: PropertyEntry = PropertyEntry("wd.warp.log.stacktraces", isRequired = false, "false")
```

Comment:
should we go more granular in our log naming? i.e `wd.warp.log.mc.stacktraces`

Reply:
cool ill rename it